x-pack/plugins/security/server/plugin.test.ts

```diff
@@ -145,6 +151,21 @@ describe('Security Plugin', () => {
 
       expect(mockCoreSetup.userProfile.registerUserProfileDelegate).toHaveBeenCalledTimes(1);
     });
+
+    it('logs the hash for the security encryption key', () => {
+      plugin.setup(mockCoreSetup, mockSetupDependencies);
+
+      const infoLogs = loggingSystemMock.collect(mockInitializerContext.logger).info;
+
+      // ToDo: not sure why this gets called twice
+      expect(infoLogs.length).toBe(2);
+      expect(infoLogs[0]).toEqual([
+        `Hashed 'xpack.security.encryptionKey' for this instance: WLbjNGKEm7aA4NfJHYyW88jHUkHtyF7ENHcF0obYGBU=`,
+      ]);
+      // expect(infoLogs[1]).toEqual([
+      //   `Hashed 'xpack.security.encryptionKey' for this instance: WLbjNGKEm7aA4NfJHYyW88jHUkHtyF7ENHcF0obYGBU=`,
+      // ]);
```

Comment:
I wasn't sure why this is getting logged twice.

Reply:
I think it is because `createConfig(...)` is called from a pipe function on an observable in `setup()`:

```js
const config$ = this.initializerContext.config.create<TypeOf<typeof ConfigSchema>>().pipe(
      map((rawConfig) =>
        createConfig(rawConfig, this.initializerContext.logger.get('config'), {
          isTLSEnabled: core.http.getServerInfo().protocol === 'https',
        })
      )
    );
```

Reply:
Ahh...ok. So in this case it's probably running for the official kibana yml and the dev yml?

Reply:
I can't 100% say w/o diving too deep, but I think it just emits the first time with its initial value on subscribe and then emits again after a replay/new(same) value.

I removed my dev yml and still get the double output:

```
[2024-07-12T08:58:00.269-04:00][INFO ][plugins.screenshotting.config] Chromium sandbox provides an additional layer of protection, and is supported for Darwin OS. Automatically enabling Chromium sandbox.
{
  loginAssistanceMessage: '',
  showInsecureClusterWarning: true,
  showNavLinks: true,
  cookieName: 'sid',
  encryptionKey: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
  session: {
    idleTimeout: Duration {
      _isValid: true,
      _milliseconds: 0,
      _days: 3,
      _months: 0,
      _data: [Object],
      _locale: [Locale]
    },
    lifespan: Duration {
      _isValid: true,
      _milliseconds: 0,
      _days: 30,
      _months: 0,
      _data: [Object],
      _locale: [Locale]
    },
    cleanupInterval: Duration {
      _isValid: true,
      _milliseconds: 3600000,
      _days: 0,
      _months: 0,
      _data: [Object],
      _locale: [Locale]
    }
  },
  secureCookies: false,
  public: {},
  authc: {
    selector: {},
    providers: {
      basic: [Object],
      token: undefined,
      saml: undefined,
      oidc: undefined,
      pki: undefined,
      kerberos: undefined,
      anonymous: undefined
    },
    http: { enabled: true, autoSchemesEnabled: true, schemes: [Array] }
  },
  audit: { enabled: false },
  experimental: { fipsMode: { enabled: false } }
}
[2024-07-12T08:58:00.637-04:00][INFO ][plugins.security.config] Hashed 'xpack.security.encryptionKey' for this instance: Lu5CspnLRLs9XdCgIhDOKd68IRC3xGRP84xTCElAviE=
[2024-07-12T08:58:00.637-04:00][WARN ][plugins.security.config] Session cookies will be transmitted over insecure connections. This is not recommended.
{
  loginAssistanceMessage: '',
  showInsecureClusterWarning: true,
  showNavLinks: true,
  cookieName: 'sid',
  encryptionKey: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
  session: {
    idleTimeout: Duration {
      _isValid: true,
      _milliseconds: 0,
      _days: 3,
      _months: 0,
      _data: [Object],
      _locale: [Locale]
    },
    lifespan: Duration {
      _isValid: true,
      _milliseconds: 0,
      _days: 30,
      _months: 0,
      _data: [Object],
      _locale: [Locale]
    },
    cleanupInterval: Duration {
      _isValid: true,
      _milliseconds: 3600000,
      _days: 0,
      _months: 0,
      _data: [Object],
      _locale: [Locale]
    }
  },
  secureCookies: false,
  public: {},
  authc: {
    selector: {},
    providers: {
      basic: [Object],
      token: undefined,
      saml: undefined,
      oidc: undefined,
      pki: undefined,
      kerberos: undefined,
      anonymous: undefined
    },
    http: { enabled: true, autoSchemesEnabled: true, schemes: [Array] }
  },
  audit: { enabled: false },
  experimental: { fipsMode: { enabled: false } }
}
[2024-07-12T08:58:00.647-04:00][INFO ][plugins.security.config] Hashed 'xpack.security.encryptionKey' for this instance: Lu5CspnLRLs9XdCgIhDOKd68IRC3xGRP84xTCElAviE=
```